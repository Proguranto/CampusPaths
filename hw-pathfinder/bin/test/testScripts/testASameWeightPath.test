# Find a path with Dijkstra's algorithm when
# the graph is acyclic and there are multiple paths
# from start to dest node with same edge weights.

CreateGraph graph6

AddNode graph6 S
AddNode graph6 1
AddNode graph6 2
AddNode graph6 D

AddEdge graph6 S 1 1.0
AddEdge graph6 S 2 1.0
AddEdge graph6 1 D 1.0
AddEdge graph6 2 D 1.0

FindPath graph6 S D
