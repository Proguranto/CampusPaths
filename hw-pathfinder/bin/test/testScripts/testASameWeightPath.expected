# Find a path with Dijkstra's algorithm when
# the graph is acyclic and there are multiple paths
# from start to dest node with same edge weights.

created graph graph6

added node S to graph6
added node 1 to graph6
added node 2 to graph6
added node D to graph6

added edge 1.000 from S to 1 in graph6
added edge 1.000 from S to 2 in graph6
added edge 1.000 from 1 to D in graph6
added edge 1.000 from 2 to D in graph6

path from S to D:
S to 1 with weight 1.000
1 to D with weight 1.000
total cost: 2.000
